openlcs/libs/common.py

```diff
@@ -100,8 +101,35 @@ def compress_source_to_tarball(dest_file, src_dir):
 
 
 def get_nvr_list_from_components(components, comp_type):
+    """
+    Get a nvr list of components.
+    """
     nvr_list = []
-    for component in components.get(comp_type):
-        nvr = "{name}-{version}-{release}".format(**component)
+    components = components.get(comp_type)
+    for component in components:
```

Comment:
It will give the error "TypeError: 'NoneType' object is not iterable" when import a source container, like "ubi8-micro-container-source-8.6-394.1".

Reply:
Updated